app/[addressOrDomain]/page.tsx

```diff
@@ -83,6 +83,8 @@ export default function Page({ params }: AddressOrDomainProps) {
   const [identity, setIdentity] = useState<Identity>();
   const [notFound, setNotFound] = useState(false);
   const [isOwner, setIsOwner] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
```

Comment:
useless space + unprecise naming, what is loading here ? I would go for `isProfileCardLoading`